src/Day05.kt

```diff
@@ -17,13 +17,12 @@ fun main() {
         var totalNiceList = 0
 
         for(string in input) {
-            when(isNice(string)) {
-                true        -> totalNiceList++
-                false       -> continue
-            }
+            if (isNice(string)) totalNiceList++
         }
 
-        return totalNiceList
+        return input.filter { isNice(it) }.count()
+
+//        return totalNiceList
     }
```

Comment:
Nothing wrong with the approach you had in place (i.e. the `for` loop and return). However, we can use a couple functional methods to simplify. This `return` statement replaces the entire `for` loop and your original `return` and compresses it down to a single line.